beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/publisher/BlockPublisherDeneb.java

```diff
@@ -64,8 +64,8 @@ void publishBlockAndBlobSidecars(
       final SignedBeaconBlock block,
       final List<BlobSidecar> blobSidecars,
       final BlockPublishingPerformance blockPublishingPerformance) {
-    blockGossipChannel.publishBlock(block);
-    blobSidecarGossipChannel.publishBlobSidecars(blobSidecars);
+    blockGossipChannel.publishBlock(block).ifExceptionGetsHereRaiseABug();
```

Comment:
So if there will be error on block publishing, no peers, or whatever, it will wrapped with "IF exception is here..."? It's not correct I think

Reply:
As you never push error higher (see `GossipHandler,gossip`), it was just my misunderstanding, we should never get exception here, right